src/Dispatcher/DebugShutdownDispatcher.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Dispatcher;
+
+use Psr\Container\ContainerInterface;
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Yiisoft\Yii\Debug\Debugger;
+use Yiisoft\Yii\Web\Event\ApplicationShutdown;
+
+class DebugShutdownDispatcher implements EventDispatcherInterface
```

Comment:
I think we can just subscribe to the event in question. Are you sure we need to implement dispatcher?